gulpfile.js

```diff
@@ -1,16 +1,30 @@
 var gulp = require( 'gulp' );
 var requireDir = require( 'require-dir' );
+var runSequence = require('run-sequence');
 
 requireDir( './gulp' );
 
-gulp.task( 'watch-compile', [ 'compile' ], function () {
+function watchAndRebuild() {
+	var watcher = gulp.watch( ['./compile/**/*'], [ 'build' ] );
+	watcher.on('change', function(event) {
```

Comment:
compile directory was auto generated by browserify and another compiling tasks, so message was always `compile/app.js was changed, running tasks...`.
watcher on `watchAndRecompile` is more better.